### ISSUE TITLE
Add support to deploy images from a remote URL

### ISSUE THREAD

**Issue by UriHerrera** [Nitrux/znx#9]
I tried deploying an ISO specifying a remote URL (HTTP, FTP) but failed to complete.

I think that it would work 

1. by downloading the ISO to `/tmp`
2. then deploy the image to the target device, and 
3. delete the ISO after the deployment has been successful.

This is currently not possible, only local ISO files can be deployed.

**Comment by luis-lavaire** [comment]
Can you give more details about the issue, @UriHerrera? The output of the command, the arguments you passed, etc.

**Comment by UriHerrera** [comment]
I tried to deploy an image using znx from a web URL like this `znx deploy $DEVICE $IMAGENAME $WEBURL` znx starts to download the file but then it fails to deploy it.

This is what I get
```
Deploying nitrux/mkiso.
cp: cannot stat 'http://88.198.66.58:8000/nitrux.iso': No such file or directory
#################### 100.0% 7402.4 kBps DONE      

Bad line - not a zsync file? ""
znx: Error: Failed to deploy nitrux/mkiso.
```

**Comment by luis-lavaire** [comment]
Oh, I see. `znx` is able to deploy ISO images that a `.zsync` file refers to. But it is a good idea to have the ability to deploy ISO images directly.

**Comment by UriHerrera** [comment]
Also, I was thinking that to speed up the download `znx` could use `axel`.

**Comment by luis-lavaire** [comment]
Solved in 5923d61.
